requirements.txt

```diff
@@ -0,0 +1,255 @@
+absl-py==1.4.0
```

Comment:
include direct dependencies only. if you're exporting from conda env, you might want to use `--from-history` param